service/src/main/java/bio/terra/catalog/api/PublicApiController.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.catalog.api;
+
+import bio.terra.catalog.config.VersionConfiguration;
+import bio.terra.catalog.model.SystemStatus;
+import bio.terra.catalog.model.VersionProperties;
+import bio.terra.catalog.service.CatalogStatusService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class PublicApiController implements PublicApi {
+  private final CatalogStatusService statusService;
+  private final VersionProperties currentVersion;
+
+  @Autowired
+  public PublicApiController(CatalogStatusService statusService,
+                             VersionConfiguration versionConfiguration) {
+    this.statusService = statusService;
+    this.currentVersion =
+        new VersionProperties()
+            .gitTag(versionConfiguration.getGitTag())
+            .gitHash(versionConfiguration.getGitHash())
+            .github(
+                "https://github.com/DataBiosphere/terra-billing-profile-manager/commit/"
```

Comment:
Looks like this was leftover from something else?

```suggestion
                "https://github.com/DataBiosphere/terra-data-catalog/commit/"
```

Reply:
Nice catch. It turns out the generated `git.properties` has this URL already so I can remove the hardcoded path here too.